contracts/estate/EstateRegistry.sol

```diff
@@ -39,6 +39,15 @@ contract EstateRegistry is Migratable, IEstateRegistry, ERC721Token, ERC721Recei
     _;
   }
 
+  modifier onlyManager(uint256 estateId) {
```

Comment:
canManageOperators?

Reply:
Hmm idk. It involves also to update content. I thought `canManage` but is super generic

Reply:
"OnlyManager" can be interpreted as "not the owner, not the operator, only the manager".

Reply:
canSetUpdateOperator